packages/jsonmodels/ledgerstate.go

```diff
@@ -484,32 +484,21 @@ type OutputMetadata struct {
 	Solid              bool                `json:"solid"`
 	SolidificationTime int64               `json:"solidificationTime"`
 	ConsumerCount      int                 `json:"consumerCount"`
-	FirstConsumer      string              `json:"firstConsumer,omitempty"`
 	ConfirmedConsumer  string              `json:"confirmedConsumer,omitempty"`
 	GradeOfFinality    gof.GradeOfFinality `json:"gradeOfFinality"`
 }
 
 // NewOutputMetadata returns the OutputMetadata from the given ledgerstate.OutputMetadata.
 func NewOutputMetadata(outputMetadata *ledgerstate.OutputMetadata) *OutputMetadata {
```

Comment:
add `confirmedConsumer TransactionID` as a parameter here instead of the need to implicitly set it.